demo/launch/pipeline_testbench.launch.py

```diff
@@ -72,34 +71,6 @@ def launch_setup(context, *args, **kwargs):
         },
     }
 
-    # Load additional OMPL-STOMP chain pipeline
-    ompl_stomp_planning_pipeline_config = {
-        "ompl_stomp": {
-            "planning_plugins": [
-                "ompl_interface/OMPLPlanner",
-                "stomp_moveit/StompPlanner",
-                "ktopt_interface/KTOptPlanner"
-            ],
-            "request_adapters": [
-                "default_planning_request_adapters/ResolveConstraintFrames",
-                "default_planning_request_adapters/ValidateWorkspaceBounds",
-                "default_planning_request_adapters/CheckStartStateBounds",
-                "default_planning_request_adapters/CheckStartStateCollision",
-            ],
-            "response_adapters": [
-                "default_planning_response_adapters/AddTimeOptimalParameterization",
-                "default_planning_response_adapters/ValidateSolution",
-                "default_planning_response_adapters/DisplayMotionPath",
-            ],
-            "planner_configs": {
-                "RRTConnectkConfigDefault": {
-                    "type": "geometric::RRTConnect",
-                    "range": 0.0,  # Max motion added to tree. ==> maxDistance_ default: 0.0, if 0.0, set on setup()}
-                }
-            },
-        }
-    }
```

Comment:
Don't you think we can still include all of these pipelines and visualise each trajectory to compare between them? I commented out the ompl_stomp just to test, but in the final version it would be nice to have all of them lying around and visualise all the trajectories in the demo.

Reply:
I think comparing it to stomp and ompl is sufficient. I'd remove it for now. We can easily re-add it later but I think for now it is mostly noise